Terminal.Gui/View/View.cs

```diff
@@ -1805,6 +1802,8 @@ public virtual void Redraw (Rect bounds)
 
 			OnDrawFrames ();
 
+			OnRenderLineCanvas ();
```

Comment:
You are not understanding how this is supposed to work (it's not all there yet, so that's understandable).

`View.OnDrawFrames` writes all borders to `View.LineCanvas`. 

Then, at a later time (any time) `View.OnRenderLineCanvas` will be called to cause the `View.LineCanvas` contents to be rendered.  The full architecture for doing this right is not in place yet. 

When I merged #2527 I had hacked things so that EVERYTHING except for those two unit test worked. 

I'd like you to see if you can fix the unit tests another way.

If you can't that's Ok. we can fix them later when more of the new architecture is in place. They cover edge cases anyway.

Reply:
Without the changes I made on Views I couldn't fix them, sorry. I'm still confused with all these new code.

Reply:
I reverted some code, but the `Menu` border need to render first for the separator drawing well.